src/views/user/landing_page.ejs

```diff
@@ -1,4 +1,8 @@
 <%- include('./navbar_v1') %>
```

Comment:
Co Ty na to, żeby sparametryzować navbar według tego czy użytkownik jest zalogowany? 

Dla zalogowanego wyświetlić przyciski "Shopping cart", "My account" oraz "Logout", a dla niezalogowanego wyświetlić tylko przycisk "Login" (i nie wyświetlać "Shopping cart" oraz "My account")?

Reply:
Zrobione.